packages/data-stores/src/site/actions.ts

```diff
@@ -57,18 +63,43 @@ export function createActions( clientCreds: WpcomClientCredentials ) {
 		}
 	}
 
+	const receiveExistingSite = ( slug: string, response: ExistingSiteDetails | undefined ) => ( {
```

Comment:
How about dropping the `Existing` infix (here and elsewhere)? (Again somewhat informed by what Calypso does.) In a general-purpose context (and `data-stores` is general purpose), I think we'll usually use these tools to check for existing sites rather than new ones.

Reply:
Great idea. I've gone through and renamed the reducer and actions to refer to `sites` instead of existingSites.